examples/README.md

```diff
@@ -8,38 +8,41 @@ md_toc github examples/README.md | sed -e "s/\s-\s/ * /"
 -->
```

Comment:
nit: I would have suggested doing the reorg in it's own commit.

Reply:
I thought it was easier to bundle, since I was already there. It was a mistake and I reverted it now. PTAL

Reply:
SG. I was even saying 2 commits 1 PR would have been easier to review. Looks good now I think.